[CUBRIDMAN-226] manual for plandump and lockdb (#523)
CUBRID/cubrid-manual: en/admin/admin_utils.rst

  -d, --drop                   drop all plans in the server's cache
  -o, --output-file=FILE       redirect output messages to FILE

>   -s, --sha1=SHA1              drop specific plan in the server's cache

Author: ```suggestion
  -s, --sha1=SHA1              drop specific plan by SHA1 in the server's cache
```

>   -s, --sha1=SHA1              drop specific plan by SHA1 in the server's cache

Reviewer: ```suggestion
  -s, --sha1=SHA1              drop specific plan identified by SHA1 from the server's cache
```

        cubrid plandump -o output.txt testdb

.. option:: -s, --sha1=SHA1

>     This option drops the specific query plans stored in the cache. ::

Author: ```suggestion
    This option drops the specific query plans by SHA1 in the server's cache. ::
```

>     This option drops the specific query plans by SHA1 in the server's cache. ::

Reviewer: ```suggestion
    This option drops specific query plans identified by their SHA1 hashes from the server's cache. ::
```

        cubrid lockdb -o output.txt testdb

.. option:: -c, --contention

>     The **-c** option displays the lock information which has contention. ::

Reviewer: ```suggestion
    The **-c** option displays information about locks that are in contention ::
```

>     The **-c** option displays the lock information which has contention. ::

Reviewer: ```suggestion
    The **-c** option displays information on locks that are in contention. ::
```

**Object lock table**

> The third section of the output of the **cubrid lockdb** includes the contents of the object lock table. It shows which client has the lock for which object in which mode, and which client is waiting for which object in which mode. The first part of the result of the object lock table shows how many objects are locked and how many lock objects are allocated to memory. CUBRID reuses lock objects for performance, but if the number of locks exceeds the **lock_escalation**, the lock objects are freed from memory at the end of the transaction.

Author: ```suggestion
The third section of the output of the **cubrid lockdb** includes the contents of the object lock table. It shows which client has the lock for which object in which mode, and which client is waiting for which object in which mode. The first part of the result of the object lock table shows how many objects are locked and how many lock objects are allocated to memory. CUBRID reuses lock objects for performance, but if the number of locks exceeds the **lock_escalation**, the lock objects are freed from memory at the end of the transaction. For more details, see :ref:`lock-parameters`.
```

> The third section of the output of the **cubrid lockdb** includes the contents of the object lock table. It shows which client has the lock for which object in which mode, and which client is waiting for which object in which mode. The first part of the result of the object lock table shows how many objects are locked and how many lock objects are allocated to memory. CUBRID reuses lock objects for performance, but if the number of locks exceeds the **lock_escalation**, the lock objects are freed from memory at the end of the transaction. For more details, see :ref:`lock-parameters`.

Reviewer: ```suggestion
The third section of the output of the **cubrid lockdb** includes the contents of the object lock table. It shows which client has the lock for which object in which mode, and which client is waiting for which object in which mode. The first part of the result of the object lock table shows how many objects are locked and how many lock objects are allocated to memory. CUBRID reuses lock objects for performance, but if the number of locks exceeds the value of **lock_escalation**, the lock objects are freed from memory at the end of the transaction. For more details, see :ref:`lock-parameters`.
```

::

    Object lock Table:

>         Current number of objects which are locked = 2001

Author: ```suggestion
        Current number of objects which are locked = 100
```

                        Start_waiting_at = Wed Feb 3 14:45:14 2016
                        Wait_for_secs = -1
                        

> The next example shows an instance of class, object OID( 2| 50| 1 ), that was inserted by transaction 1 which holds **X_LOCK** on the object. The class has a unique index and the key of inserted instance is about to be modified by transaction 2, which is blocked until transaction 1 is completed. The **-c** option displays the lock information which has contention.

Reviewer: ```suggestion
The next example shows an instance of class, object OID( 2| 50| 1 ), that was inserted by transaction 1 which holds **X_LOCK** on the object. The class has a unique index and the key of inserted instance is about to be modified by transaction 2, which is blocked until transaction 1 is completed. The **-c** option displays only the information on locks that are involved in contention.
```
